vendor/vendor.json

```diff
@@ -114,6 +114,12 @@
 			"revision": "99f16d856c9836c42d24e7ab64ea72916925fa97",
 			"revisionTime": "2017-03-08T15:04:45Z"
 		},
+		{
+			"checksumSHA1": "alYroN0n+3NDlMXZeiC6d/LG764=",
+			"path": "google.golang.org/grpc",
+			"revision": "9cb02b885b41e0db982e23d6e51a24ea465dec2e",
+			"revisionTime": "2017-06-26T23:20:44Z"
+		},
```

Comment:
Please don't update the vendor source code from example code.

Reply:
got it, but if have not this, ci will fail.

Reply:
and I have removed it.